packages/jest-util/src/__tests__/create_process_object.test.js

```diff
@@ -5,14 +5,14 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import EventEmitter from 'events';
 import createProcessObject from '../create_process_object';
 
 it('creates a process object that looks like the original one', () => {
   const fakeProcess = createProcessObject();
 
-  // "process" inherits from EventEmitter through the prototype chain.
-  expect(fakeProcess instanceof EventEmitter).toBe(true);
```

Comment:
Hah, I think this is a bug actually. You are creating the process object with EventEmitter from the parent but it should use `runtime._requireNativeModule('events')` so that this test still passes. Changing this test isn't the right fix here imho.

Reply:
Exactly; I explained that to @thymikee before. The issue here is that in order to get the child `EventEmitter` I need to execute in the context `require('events').EventEmitter`. Unfortunately at this point neither the context, nor the require implementation are ready; so it's too complex to add the change in this PR.

Since I just re-did the `process` object about a week ago, and before it did not inherit from `EventEmitter` either, I think we are more or less safe to do this.

However, at the core of the issue, you are both right, the check should pass 🙂

Reply:
In that case let's change where we are initializing process then? It seems like that should happen later or lazily. You could make `process` a lazy getter that, when first accessed, overwrites itself by doing `global.process = process`. What do you think about that? We just need to make sure there is nothing in `events` or its dependencies that depends on process which would produce a circular dependency.

Reply:
I tried making this work, but I ended up with a V8 crash (see below). I think we should revisit it later; but I should be kept this out of the scope of this PR.

```
FATAL ERROR: v8::ToLocalChecked Empty MaybeLocal.
 1: node::Abort() [/Users/.../.nvm/versions/node/v8.9.1/bin/node]
 2: node::FatalException(v8::Isolate*, v8::Local<v8::Value>, v8::Local<v8::Message>) [/Users/.../.nvm/versions/node/v8.9.1/bin/node]
 3: v8::V8::ToLocalEmpty() [/Users/.../.nvm/versions/node/v8.9.1/bin/node]
 4: node::(anonymous namespace)::ContextifyScript::RunInContext(v8::FunctionCallbackInfo<v8::Value> const&) [/Users/.../.nvm/versions/node/v8.9.1/bin/node]
 5: v8::internal::FunctionCallbackArguments::Call(void (*)(v8::FunctionCallbackInfo<v8::Value> const&)) [/Users/.../.nvm/versions/node/v8.9.1/bin/node]
 6: v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [/Users/.../.nvm/versions/node/v8.9.1/bin/node]
 7: v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [/Users/.../.nvm/versions/node/v8.9.1/bin/node]
 8: 0x2fbc95e0463d
 9: 0x2fbc95ef4048
```

Reply:
Can you file a task to fix this?